fastlane_core/spec/config_item_spec.rb

```diff
@@ -0,0 +1,43 @@
+describe FastlaneCore do
+  describe FastlaneCore::ConfigItem do
+    describe "ConfigItem input validation" do
+      it "doesn't raise an error if everything's valid" do
+        result = FastlaneCore::ConfigItem.new(key: :foo,
+                                     short_option: "-f",
+                                     description: "foo")
+        expect(result).to be_a(FastlaneCore::ConfigItem)
```

Comment:
This doesn't seem worth testing. If we call a constructor on a class and have the result not `be_a` thing of that class, we're in trouble :smile: